ec2utils/ec2deprecateimg/ec2deprecateimg

```diff
@@ -0,0 +1,205 @@
+    metavar='CONFIG_FILE'
+)
+depImgIdentifier = argparse.add_mutually_exclusive_group(required=True)
+depImgIdentifier.add_argument(
+    '--image-id',
+    dest='depImgID',
+    help='The AMI ID of the image to be deprecated (Optional)',
+    metavar='AMI_ID'
+)
+depImgIdentifier.add_argument(
+    '--image-name',
+    dest='depImgName',
+    help='The image name of the image to be deprecated (Optional)',
+    metavar='IMAGE_NAME'
+)
+hlpMsg = 'An image name fragment to match the image name of the image to be '
+hlpMsg += 'deprecated (Optional)'
+depImgIdentifier.add_argument(
+    '--image-name-frag',
+    dest='depImgNameFrag',
+    help=hlpMsg,
+    metavar='IMAGE_NAME_FRAGMENT'
+)
+hlpMsg = 'A regular expression to match the image name of the image to be '
+hlpMsg += 'deprecated (Optional)'
+depImgIdentifier.add_argument(
+    '--image-name-match',
+    dest='depImgNameMatch',
+    help=hlpMsg,
+    metavar='REGEX'
+)
+argparse.add_argument(
+    '--image-virt-type',
+    choices=['hvm','para'],
+    dest='virtType',
+    help='The virtualization type of the image to be deprecated (Optional)',
+    metavar='VIRT_TYPE'
+)
+argparse.add_argument(
+    '--public-only',
+    action='store_true',
+    default=False,
+    dest='publicOnly',
+    help='Only consider images that are public (Optional)'
+)
+replImgIdentifier = argparse.add_mutually_exclusive_group(required=True)
+hlpMsg = 'The AMI ID of the image used as a replacement for the image(s) '
+hlpMsg += 'being deprecated (Optional), one of --replacement-id or '
+hlpMsg += '--replacement-name is required'
+replImgIdentifier.add_argument(
+    '--replacement-id',
+    dest='replImgID',
+    help=hlpMsg,
+    metavar='AMI_ID'
+)
+hlpMsg = 'The name of the image used as a replacement for the image(s) '
+hlpMsg += 'being deprecated (Optional), one of --replacement-id or '
+hlpMsg += '--replacement-name is required'
+replImgIdentifier.add_argument(
+    '--replacement-name',
+    dest='replImgName',
+    help=hlpMsg,
+    metavar='IMAGE_NAME'
+)
+hlpMsg = 'An image name fragment to match the image name of the image to be '
+hlpMsg += 'deprecated (Optional)'
+replImgIdentifier.add_argument(
+    '--replacement-name-frag',
+    dest='replImgNameFrag',
+    help=hlpMsg,
+    metavar='IMAGE_NAME_FRAGMENT'
+)
+hlpMsg = 'A regular expression to match the image name of the image to be '
+hlpMsg += 'deprecated (Optional)'
+replImgIdentifier.add_argument(
+    '--replacement-name-match',
+    dest='replImgNameMatch',
+    help=hlpMsg,
+    metavar='REGEX'
+)
+hlpMsg = 'Comma separated list of regions for publishing, all integrated '
+hlpMsg += 'region sif not given (Optional)'
+argparse.add_argument(
+    '-r', '--regions',
+    dest='regions',
+    help=hlpMsg,
+    metavar='EC2_REGIONS'
+)
+argparse.add_argument(
+    '--verbose',
+    dest='verbose',
+    help='Enable on verbose output'
+)
+argparse.add_argument(
+    '--version',
+    dest='version',
+    help='Program version'
+)
```

Comment:
Nope, as is --version is treated just like any other argument, thus we can implement custom behavior. The custom behavior allows us to unify the version for setup.py, the script, and version checking in the Makefile